File: app/src/main/java/com/example/whattodo/ItemEvent_Activity.java
package com.example.whattodo;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

public class ItemEvent_Activity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_eventitem);

        getSupportFragmentManager().beginTransaction().replace(R.id.fragment_eventitem, new ItemEvent_Fragment()).commit();
    }
}

File: app/src/main/java/com/example/whattodo/ItemEvent_Activity.kt
package com.example.whattodo

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity

class ItemEvent_Activity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_eventitem)
        supportFragmentManager.beginTransaction().replace(R.id.fragment_eventitem, ItemEvent_Fragment()).commit()
    }
}